### PR TITLE
Add support for HLS in the Docker container.

### PR DESCRIPTION
**Pull request by BenLubar**

### CODE REVIEW

File: support/docker/production/Dockerfile.stretch [hunk 1]
```diff
@@ -1,4 +1,5 @@
-FROM node:10-stretch-slim
+# use buster for new enough ffmpeg
```
Comment (rigelk) [inline support/docker/production/Dockerfile.stretch]: Please remove the comment, and instead specifiy that kind of extraneous information in the body of your PR.

**Comment by Chocobozzz** [comment]
Thanks @BenLubar!

File: support/docker/production/Dockerfile.stretch (hunk 1 above)
Comment (BenLubar) [reply support/docker/production/Dockerfile.stretch]: I added it as a comment because the file is named stretch

**Comment by rigelk** [comment]
@BenLubar could you rename the file to end in "buster"? That would make more sense.

**Comment by Chocobozzz** [comment]
I'll do it

**Pull request merged**